sns_ik_lib/src/sns_acc_ik_base.cpp

```diff
@@ -226,40 +168,36 @@ SnsAccIkBase::ExitCode SnsAccIkBase::solve(const Eigen::MatrixXd& J, const Eigen
     // Saturate the most critical joint
     W(jntIdx, jntIdx) = 0.0;
     jointIsFree[jntIdx] = false;
-    if ((*ddq)(jntIdx) > ddqUpp_(jntIdx)) {
-      ddqNull(jntIdx) = ddqUpp_(jntIdx);
-    } else if ((*ddq)(jntIdx) < ddqLow_(jntIdx)) {
-      ddqNull(jntIdx) = ddqLow_(jntIdx);
+    if ((*ddq)(jntIdx) > (getUpp())(jntIdx)) {
```

Comment:
A prime example of where the naming changes that I outlined above would make this code easier to read.

Reply:
I agree.